http/src/main/resources/reference.conf

```diff
@@ -66,9 +66,10 @@ gce {
   ]
   runtimeDefaults {
     machineType = "n1-standard-4"
-    diskSize = 100
+    diskSize = 20
```

Comment:
Mm.. 20 seems low, no? Why not 50 or 100?

Reply:
I guess 50G will be equivelant to what it was before.